.eslintrc

```diff
@@ -4,8 +4,18 @@
     "browser": true,
     "jest": true
   },
+  "parserOptions": {
+    "ecmaVersion": 7,
```

Comment:
use `"ecmaVersion": 6,`

Reply:
Let me make that change,  thank you.

Reply:
@justmesam  I have implemented the changes.